README.md

```diff
@@ -1,2 +1,75 @@
-# baby-bliss-bot
-An exploratory research to generate new Bliss vocabulary
+# Baby Bliss Bot
+
+An exploratory research to generate new Bliss vocabulary.
+
+[Bliss language](https://www.blissymbolics.org/) is an Augmentative and Alternative Communication (AAC) language
+used by individuals with severe speech and physical impairments around the world, but also by others for language
+learning and support, or just for the fascination and joy of this unique language representation. It is a semantic
+graphical language that is currently composed of more than 5000 authorized symbols - Bliss-characters and Bliss-words.
+It is a generative language that allows its users to create new Bliss-words as needed.
+
+The generation of new vocabulary situates in the context of an exploratory research that could leverage from emerging
```

Comment:
Minor edit: "We are exploring the generation of new Bliss vocabulary using emerging AI techniques, including Large Language Models (LLM), OCR, and other models for text generation and completion."